x/projects/keeper/creation.go

```diff
@@ -10,12 +10,16 @@ import (
 
 // add a default project to a subscription, add the subscription key as
 func (k Keeper) CreateAdminProject(ctx sdk.Context, subscriptionAddress string, totalCU uint64, cuPerEpoch uint64, providers uint64, vrfpk string) error {
```

Comment:
just pass the policy instead of the parameters

Reply:
In the add project to subscription PR, I did that. So after all of my PRs will be merged, it'll be fixed